packages/server/README.md

```diff
@@ -1,23 +1,28 @@
 # Installing
 
-## Configuration
+## Requirements
 
-First copy `.env-sample` to `.env` and edit the values in there:
+- Docker Engine run Joplin Server. See [Install Docker Engine](https://docs.docker.com/engine/install/) for steps to install Docker Engine for your operating system.
+- Docker Compose is required to store item contents (notes, tags, etc.) if PostgreSQL is not used. See [Install Docker Compose](https://docs.docker.com/compose/install/) for steps to install Docker Compose for your operating system.
 
-- `APP_BASE_URL`: This is the base public URL where the service will be running. For example, if you want it to run from `https://example.com/joplin`, this is what you should set the URL to. The base URL can include the port.
-- `APP_PORT`: The local port on which the Docker container will listen. You would typically map this port to 443 (TLS) with a reverse proxy.
+## Configure Docker for Joplin Server
 
-## Running the server
+1. Copy `.env-sample` (located [here](https://github.com/laurent22/joplin/blob/dev/.env-sample)) to the location of your Docker configuration files. Example: /home/[user]/docker
+2. Rename the file `.env-sample` to `.env`.
 
-To start the server with default configuration, run:
+
+## Test Starting the Server
+
+1. Run the following command to test starting the server using the default configuration:
 
 ```shell
 docker run --env-file .env -p 22300:22300 joplin/server:latest
```

Comment:
Please restore the configuration section. There wasn't much in there, so I don't think it needs to be split into two sections.